Azure/azure-cli
# [ARM] `az stack`: Add No Wait Support for Stack Create Commands

**Related command**

az stack sub create 
az stack group create
az stack mg create 

**Description**

The parameter --no-wait was added to all of the above commands. This allows customers not to wait on the finishing of a stack deployment. 

**Testing Guide**
The show command is to check whether or not the stack completed deployment. 


az stack sub create -n stackname -f templatefilepath --deny-settings-mode "None"  -l westus2 --no-wait
az stack sub show -n stackname 


az stack group create -n stack -f templatefilepath.json --deny-settings-mode "None"  -g resourcegroupname --no-wait
az stack group show -g resourcegroupname -n stackname


az stack mg create -n stackname  -m ManagmentGroupName  --location westus2 --template-file templatefilepath.json  --dm  "None"   --no-wait
az stack mg show -n stack name  -m ManagmentGroupName 

**History Notes**


[ARM] `az stack sub create`: Add no wait support
[ARM] `az stack mg create`: Add no wait support
[ARM] `az stack group create`: Add no wait support

---

This checklist is used to make sure that common guidelines for a pull request are followed.
